bugsnag-android-core/src/main/java/com/bugsnag/android/AppDataCollector.kt

```diff
@@ -49,12 +64,55 @@ internal class AppDataCollector(
         )
     }
 
+    @SuppressLint("SwitchIntDef")
+    @Suppress("DEPRECATION")
+    private fun getProcessImportance(): String? {
+        try {
+            val appInfo = ActivityManager.RunningAppProcessInfo()
+            if (appInfo.pid == 0) {
+                return null
+            }
```

Comment:
This will early exit 100% of the time as `appInfo.pid` will always be `0` at this point